manifest-json-schema.json

```diff
@@ -0,0 +1,115 @@
+{
+  "title": "otp-runner manifest.json",
+  "description": "A manifest for helping run OpenTripPlanner.",
+  "properties": {
+    "buildConfigJSON": {
+      "type": "string",
+      "description": "The raw contents to write to the build-config.json file."
+    },
+    "buildGraph": {
+      "type": "boolean",
+      "description": "If true, run OpenTripPlanner in build mode",
+      "default": true
+    },
+    "buildLogFile": {
+      "type": "string",
+      "description": "The path where the build logs should be written to.",
+      "default": "/var/log/otp-build.log"
+    },
+    "graphObjUrl": {
+      "type": "string",
+      "description": "A url where the Graph.obj should be downloaded from for server-only runs. If `uploadGraph` is set to true, this value must be an s3 url that can be uplaoded to."
+    },
+    "graphsFolder": {
+      "type": "string",
+      "description": "The folder where the graphs should be stored.",
+      "default": "/var/otp/graphs"
+    },
+    "gtfsAndOsmUrls": {
+      "type": "array",
+      "description": "An array of GTFS and OSM urls that should be downloaded.",
+      "items": {
+        "type": "string",
+        "description": "Either a url or s3 url."
+      }
+    },
+    "jarFile": {
+      "type": "string",
+      "description": "The full path to the OTP jar file.",
+      "default": "/opt/otp-1.4.0-shaded.jar"
+    },
+    "jarUrl": {
+      "type": "string",
+      "description": "A url where the OTP jar can be downloaded from.",
+      "default": "https://repo1.maven.org/maven2/org/opentripplanner/otp/1.4.0/otp-1.4.0-shaded.jar"
+    },
+    "otpRunnerLogFile": {
+      "type": "string",
+      "description": "The path where the otp-runner logs should be written to.",
+      "default": "/var/log/otp-runner.log"
+    },
+    "prefixLogUploadsWithInstanceId": {
+      "type": "boolean",
+      "description": "If true, will obtain the ec2 instance ID and prefix the otp-runner and otp-server log files with this instance ID when uploading to s3.",
+      "default": false
+    },
+    "routerConfigJSON": {
+      "type": "string",
+      "description": "The raw contents to write to the router-config.json file."
+    },
+    "routerName": {
+      "type": "string",
+      "description": "The name of the OTP router.",
+      "default": "default"
+    },
+    "runServer": {
+      "type": "boolean",
+      "description": "If true, run OTP as a server.",
+      "default": true
+    },
+    "s3UploadBucket": {
+      "type": "string",
+      "description": "The base path of an s3 bucket where files will be uploaded to."
```

Comment:
Not sure this field name accurately reflects what it is. "bucket" implies the bucket name, not the S3 path. Also, an example format should be included in description (e.g., `s3://path/to/folder`).